test/NuGet.Core.Tests/NuGet.Versioning.Test/NuGetVersionTest.cs

```diff
@@ -322,11 +322,11 @@ public void ToStringFromStringFormat(string versionString, string specialVersion
             // Arrange 
             var version = new Version(versionString);
 
-            // Act
+               // Act
```

Comment:
`  ` space added here. Now not aligning.

Reply:
no idea how this happened... thanks!